ts/parts-more/Polar.ts

```diff
@@ -78,7 +82,7 @@ declare global {
                 high: number,
                 start: number,
                 end: number
-            ): SVGPathArray;
+            ): SVGPathArray|SVGAttributes;
```

Comment:
I don't see related changes in code

Reply:
This is added because the `polarArc` function can return not only path array but also attributes specific for arcs (`x`, `y`, `r`, `innerR`, `start`, `end`). The reason why I have changed it to work that way is explained in the comment below (question about the animation).

Reply:
Right now the `polarArc` returns only `SVGAttributes` object.

Reply:
Shouldn't `SVGPathArray` be removed from the above then?

Reply:
Ahh yes, I have changed the function itself but forgot about getting rid of the return type. Corrected, thanks!